docs/home/whats-new-2-16.md

```diff
@@ -25,6 +25,8 @@ date: 2023-Dec-4
 
     For more information, see [Workloads Overview](../admin/workloads/README.md).
 
+* <!-- RUN-15456/RUN-15457 - Add a default auto deletion time after completion -->Changed the workload default *auto deletion time after completion time* value from *Never* to *90 days*. This ensures that customer environments will be cleaned from clutter and free resources for other workloads. The default setting can be edited and adjusted by administrators using policies or by researchers at the time of submitting their workloads.
```

Comment:
```suggestion
* <!-- RUN-15456/RUN-15457 - Add a default auto deletion time after completion -->Changed the workload default *auto deletion time after completion time* value from *Never* to *90 days*. This ensures that customer environments will be cleaned from cluttering old data. Since this is only a default value and not a locked value, Researchers can override this value at the time of submitting their workloads. In addition, administrators can change the default to be of a different value (for example, change it from 90 days to 30 days/"never"/etc.) and/or lock this value using workload policies so it won't be editable for the researchers.
```

Reply:
```suggestion
* <!-- RUN-15456/RUN-15457 - Add a default auto deletion time after completion -->Changed the workload default *auto deletion time after completion* value from `Never` to `90 days`. This ensures that environments will be cleaned from old data. This field is editbale by default, allowing researchers the ability to change the value using the workload submission form. In addition, using workload policies, administrators have the option to lock access to this value so that researchers can not edit it, or increase, decrease the default value, or set the value to `never`. 
```